ci/gitlab/views.py

```diff
@@ -109,7 +109,7 @@ def process_pull_request(user, data):
         # anymore so we won't be able to fill out the full PullRequestEvent
         # (since we need additional API calls to get all the information we need).
         # So just close this manually.
-        close_pr(attributes['target']['namespace'], attributes['target']['name'], pr_event.pr_number, user.server)
+        close_pr(attributes['target']['path_with_namespace'].split('/')[0], attributes['target']['name'], pr_event.pr_number, user.server)
```

Comment:
GItLab API no longer has a designated field to retrieve 'userid'. In the entire payload available, the only mention of user id, is in the form of 'userid/repo'.

Reply:
Hmm, that seems like a rather big change on their part. I seem to recall we use the user ID on the Civet side API as the Github and Bitbucket APIs have the concept of a user ID.

Reply:
I know right!?

I doubt it's something HPC is mangling somehow. But indeed. When a PR triggers a build, the webhook no longer contains a user's userid anywhere in it (except in the form of userid/repo). I dislike needing to split on slash, but... I no no other way.

 This sort of change is what caused us to stop supporting BitBucket as well: [reference](https://developer.atlassian.com/cloud/bitbucket/bitbucket-api-changes-workspaces/).

Reply:
Maybe it is time to rewrite Civet to take into account all of these API changes! :)

Reply:
Now that I have a better understanding of what's going on... I'll certainly look into it!